frontend/kick-covid-19/src/pages/Blog.js

```diff
@@ -21,11 +21,14 @@ class Blog extends React.Component {
   }
 
   render() {
+    console.log(
+      `[DEBUG] this.state.posts = ${JSON.stringify(this.state.posts, null, 2)}`
+    );
```

Comment:
you can leave this if you want, but normally you'll want to delete logs before you merge